tests/conftest.py

```diff
@@ -9,3 +9,4 @@ def pytest_addoption(parser):
     parser.addoption("--charm-file", action="store")
     parser.addoption("--test-flask-image", action="store")
     parser.addoption("--test-db-flask-image", action="store")
+    parser.addoption("--django-app-image", action="store")
```

Comment:
Previously I think for flask we removed this, or is it the equivalent of `--test-flask-image`?